core/src/main/python/wlsdeploy/tool/compare/model_comparer.py

```diff
@@ -428,16 +428,23 @@ def _finalize_folder(self, current_folder, past_folder, change_folder, location)
         if location is not None:
             folder_path = location.get_model_folders()
 
+        app_lib_attributes = [
```

Comment:
Can we just copy every attribute in current_folder to change_folder that's not already present?
Or is there a reason we have to limit it to this subset of attributes?
It would be nice not to have to list all these attributes in this code.

Reply:
Yes, I also found logic flaw.